nodejs/awsx/package.json

```diff
@@ -11,8 +11,8 @@
     "homepage": "https://pulumi.io",
     "repository": "https://github.com/pulumi/pulumi-awsx",
     "dependencies": {
-        "@pulumi/pulumi": "^1.9.1",
-        "@pulumi/aws": "^1.0.0",
+        "@pulumi/pulumi": "^1.9.1 || ^2.0.0",
```

Comment:
why have both versions here? why not just go straight for 2.0 and we can release fixes from the last tag?

Reply:
This is a higher-level abstraction library, so it seems more odd to me to force future use onto 2.0.0. There will be some set of people using aws-1.x, in which case, they will also be using pulumi-1.x, so we should allow both.

@lukehoban I think you were also thinking of `awsx` as supporting both provider versions, but it'd be good to get your opinion if you feel otherwise.